src/image_bios.cpp

```diff
@@ -39,11 +44,13 @@ static constexpr auto goldenFlashChip = 1;
 #endif // GOLDEN_FLASH_SUPPORT
 
 static constexpr auto gpioOwner = "fwupdate";
+#ifdef USE_PCA9698_OEPOL
 static constexpr auto pca9698Bus = "/dev/i2c-11";
 static constexpr auto pca9698Addr = 0x27;
 static constexpr auto pca9698InputReg = 0x00;
 static constexpr auto pca9698ModeReg = 0x2a;
 static constexpr auto pca9698OEPolBit = 0x01;
```

Comment:
This looks like hard-coded binding to a specific output pin. Isn't it?
If so, I believe it must be extracted into an option, just like `PCH_POWER_PIN`.

Reply:
My bad. Read it wrong. It's an OE polarity, not output polarity as I thought. Don't do anything, everything is fine here.